hawkular-wildfly-agent/src/test/resources/org/hawkular/agent/monitor/extension/subsystem.xml

```diff
@@ -16,9 +16,9 @@
     limitations under the License.
 
 -->
+<!-- default value omitted: apiJndiName="java:global/hawkular/agent/api" -->
```

Comment:
OK, I'll remove the comments. I was not sure you'll ack this strategy of removing defaults.